core-processor/src/processing.rs

```diff
@@ -178,6 +178,7 @@ where
 }
 
 enum ProcessErrorCase {
+    WaitInit,
```

Comment:
extend NonExecutable case with options: if its exited/inactive (due to rent, for example) or not yet initialised

Reply:
What for?